run.py

```diff
@@ -131,7 +161,17 @@ def forbidden(error):
 
 
 @app.before_request
-def before_request():
+def before_request():   
+    '''
```

Comment:
There's no need to document those parameters. A reader of the code will see them anyway.